spec/posts_spec.rb

```diff
@@ -18,8 +16,21 @@
       expect(client.posts.get_published(1)).to eq('response')
     end
 
-    it 'should work with special characters' do
-      expect { client.posts.get_published('1 post') }.to_not raise_error(URI::InvalidURIError)
+    context 'with special characters' do
+      before(:all) do
+        stub_request(:get, 'http://cortex.dev/api/v1/posts/feed/1%20post?access_token=123')
```

Comment:
instead of this, try:

```rb
let!(:stubbed_request) { stub_request(:get, 'http://cortex.dev/api/v1/posts/feed/1%20post?access_token=123') }

expect(stubbed_request).to have_been_made.once
```

The ! on let ensures it stubs, and storing a reference means you don't have to change the stub twice if you change your pattern for some reason.